lab_1/main.py

```diff
@@ -12,8 +11,16 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
-
+    if type(text) == str:
+        text = text.replace('\n',' ')
+        text = text.lower()
```

Comment:
Usually, there are checks and body function - being separated.
And body function should be on the same level with checks.